package exception;

public class TestException4 {
public static void main(String[] args) {
String name = null;
int len = name.length();
System.out.println("length = " + len);
}
}